android/capacitor/src/main/java/com/getcapacitor/BridgeWebViewClient.java

```diff
@@ -92,4 +93,19 @@ public void onReceivedHttpError(WebView view, WebResourceRequest request, WebRes
             view.loadUrl(errorPath);
         }
     }
+
+    @Override
+    public boolean onRenderProcessGone(WebView view, RenderProcessGoneDetail detail) {
+        super.onRenderProcessGone(view, detail);
+        boolean result = false;
+
+        List<WebViewListener> webViewListeners = bridge.getWebViewListeners();
+        if (webViewListeners != null) {
+            for (WebViewListener listener : bridge.getWebViewListeners()) {
+                result = result || listener.onRenderProcessGone(view, detail);
```

Comment:
Ideally this would be:
```suggestion
                bool listenerResult = listener.onRenderProcessGone(view, detail);
                result = result || listenerResult;
```

Reply:
or maybe like this
```suggestion
                result = listener.onRenderProcessGone(view, detail) || result;
```
I like the short version better than declaring a bool on every for iteration, and I think this way will not "short circuit" since the listener check is on the left.

Reply:
Agreed! That's definitely neater @jcesarmobile 🙂

Reply:
Thanks for the catch on the short circuiting!